### ISSUE TITLE
Monte Carlo vs. Reinforced Learning

### ISSUE THREAD

**Issue by tybeady** [brs80/blackJack#1]
opinions?

I think Reinforced Learning could be used to also include "betting" into the game. As in, we use 1 deck of cards and a run of the game would involve the current AI trying to maximize its money through that single deck of cards.

This should be rather interesting to see how good it can do, and from what I can find it should be able to do pretty well.

**Comment by brs80** [comment]
Yeah the AI has to do more with betting and less with deciding to hit or pass as I originally thought.
